modules/oxide/src/less/theme/content/math/math.less

```diff
@@ -0,0 +1,13 @@
+//
+// Math
+//
+
+@math-margin: 16px 0 16px 0;
+
+/* stylelint-disable selector-type-no-unknown */
```

Comment:
Our stylelint is _very_ old, upgrading it is the first thing to try to fix this. I think I was going to do it as a part of switching oxide to the latest `less` version, and from gulp to grunt, but that fell through the cracks in the lead-up to 7.0